assets/templates/forgot-password.html

```diff
@@ -103,6 +103,12 @@ <h1>Suma contactos a tus Listas con Formularios personalizados</h1>
   <!-- build:js:dist js/app.js -->
   <script src="./js/main.js"></script>
   <!-- /build -->
+  <script type="text/javascript">
+    window.onload=function() {
```

Comment:
This can't be avoided, can it?

Reply:
The module needs a way to be instantiated; I guess its a good idea not to include the initialization in the library and make it whenever its needed instead, @cbernat

Reply:
What do you think about making doing something to automatically detect these elements and apply the behavior?